MekHQ/src/mekhq/campaign/CampaignOptions.java

```diff
@@ -4382,14 +4438,12 @@ public static CampaignOptions generateCampaignOptionsFromXml(Node wn, Version ve
         return retVal;
     }
 
-    //region Migration
     /**
      * This is annoyingly required for the case of anyone having changed the surname weights.
      * The code is not nice, but will nicely handle the cases where anyone has made changes
-     * @param retVal the return CampaignOptions
      * @param values the values to migrate
      */
-    private static void migrateMarriageSurnameWeights(CampaignOptions retVal, String... values) {
+    public void migrateMarriageSurnameWeights47(final String... values) {
```

Comment:
Is the 47 intentional?

Reply:
It is, because it's one of the two migrations for it. This one occurred in 0.47, hence the new name.